config.js

```diff
@@ -7,7 +7,10 @@ const homeDir = require('home-dir')();
 nconf
   .argv() // take args first
   .env() // then env vars
-  .use('file', { file: path.join(homeDir, '.plunk_config.json') }); // then read from file
```

Comment:
This is needed for explicit addition of default parameters instead of hardcoding them (see `defaults.json`). User can still override his/her configs in .plunk_config.json. Added comment to readme for clarity.